src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -40,7 +40,7 @@ public static int Start(RunspaceConfiguration configuration, string bannerText,
 
         /// <summary>
         /// 
-        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace
+        /// Entry point in to ConsoleShell. This method is called by the MSHManagedEntrace (SP!)
```

Comment:
You can replace it with "This method is called from the native or managed exe host application".